snuba/processor.py

```diff
@@ -38,7 +47,7 @@ class MessageProcessor(ABC):
 
     @abstractmethod
     def process_message(
-        self, message: Mapping[str, Any], metadata: KafkaMessageMetadata
+        self, message: Any, metadata: KafkaMessageMetadata
```

Comment:
This was a mistake earlier, a processor can operate on a message that can be of any format